### PR TITLE
Add cookie and privacy policies.

### PR DESCRIPTION
**Pull request by sachithKay**
Screen Shot 2022-01-25 at 5 09 17 PM
Screen Shot 2022-01-25 at 5 09 31 PM
Screen Shot 2022-01-25 at 5 09 49 PM

Fixes: https://github.com/wso2/micro-integrator/issues/2293

### CODE REVIEW

**Review by Chamodii — APPROVED** [review]
LGTM

File: monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/auth/Login.js [hunk 1]
```diff
@@ -158,6 +171,17 @@ function Login(props){
                             control={<Checkbox value="remember" color="primary" />}
                             label="Remember me"
                         />
+                        <Box mt={4} className={classes.policyBox}>
+                            <Typography variant="body2" style={{fontSize:13}}>
+                                We use browser cookies to track your session to give better experience.
```
Comment (samgnaniah) [inline monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/auth/Login.js:176]:
```suggestion
                                Browser cookies are used to track your session and provide a better experience.
```

File: monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/policies/CookiePolicy.js [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+                WSO2 Micro Integrator Dashboard monitors running Micro Integrator instances (Single or Cluster
+                Mode) and facilitates
+                performing various management and administration tasks related to deployed artifacts.
+            </Typography>
+
+            <h2 id="cookie-policy">Cookie Policy</h2>
+            <Typography>
+                WSO2 Micro Integrator Dashboard uses cookies so that it can provide the best user experience for
+                you and identify you for security purposes. If you disable cookies, some of the services will
+                (most probably) be
```
Comment (samgnaniah) [inline monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/policies/CookiePolicy.js:58]: Let's avoid "most probably" here. I think this was discussed elsewhere.

File: monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/policies/CookiePolicy.js [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+            <Typography>
+                WSO2 Micro Integrator Dashboard stores and retrieves information on your browser using cookies.
+                This information is used to provide a better experience. Some cookies serve the primary purposes
+                of allowing a user to
+                log in to the system, maintaining sessions, and keeping track of activities you do within the
+                login session.
+            </Typography>
+            <Typography>
+                The primary purpose of some cookies used in WSO2 Micro Integrator Dashboard is to personally
+                identify you as this is the main function of the WSO2 Identity Server. However the cookie
```
Comment (samgnaniah) [inline monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/policies/CookiePolicy.js:73]:
```suggestion
                identify you as this is the main function of the WSO2 Identity Server. However, the cookie
```

File: monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/policies/CookiePolicy.js [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+                lifetime ends once your session
+                ends i.e., after you log-out, or after the session expiry time has elapsed.
+            </Typography>
+            <Typography>
+                Some cookies are simply used to give you a more personalised web experience and these cookies
```
Comment (samgnaniah) [inline monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/policies/CookiePolicy.js:78]:
```suggestion
                Some cookies are simply used to give you a more personalized web experience and these cookies
```

**Review by arunans23 — APPROVED** [review]

File: monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/policies/CookiePolicy.js (hunk 2 above)
Comment (sachithKay) [reply monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/policies/CookiePolicy.js]: removed the entire section

**Pull request merged**